bcda/router.go

```diff
@@ -41,8 +41,8 @@ func NewDataRouter() http.Handler {
 	r := chi.NewRouter()
 	m := monitoring.GetMonitor()
 	r.Use(auth.ParseToken, logging.NewStructuredLogger(), HSTSHeader, ConnectionClose)
-	r.With(auth.RequireTokenAuth, auth.RequireTokenACOMatch).
-		Get(m.WrapHandler("/data/{jobID}/{acoID}.ndjson", serveData))
+	r.With(auth.RequireTokenAuth, auth.RequireTokenJobMatch).
+		Get(m.WrapHandler("/data/{jobId}/{fileName}", serveData))
```

Comment:
If the aim is to be consistent with the way we write "jobId"/"jobID", the latter is preferred in Go. Can we change it for the `/jobs/` path on line 29 and keep it `jobID` here?

Reply:
Not easily.  This syntax puts it in as a URLParam so I would need somethign wonky like:
jobID := chi.URLParam(r, "jobId")
if jobID == nil{
jobID := chi.URLParam(r, "jobID")
}
which seems worse.  If we are going to do anything I would like to refactor it all to jobID.  I started that process but it felt like a rabbit hole and abandoned it.  If you support it I am happy to take the time to do it.

Reply:
It'd be great to make this consistent/use go's convention, but wow are there are a lot of places in the code and tests that this touches. Might be better to backlog a ticket for this, as long as there's no imminent harm in leaving as-is.

Reply:
This has been refactored